uaa/src/main/webapp/resources/predix/stylesheets/application.css

```diff
@@ -2878,19 +2878,14 @@ ol.linenums {
 }
 
 .top-header {
-    position: fixed;
-    top: 0;
-    right: 0;
-    left: 0;
```

Comment:
Why was the fixed position css removed?

Reply:
We removed this so the styling is mobile friendly for the banner. Otherwise image goes under the banner on a mobile device.